values.yaml

```diff
@@ -139,7 +143,9 @@ global:
   loggingSidecar:
     enabled: false
     name: sidecar-log-consumer
-    image: quay.io/astronomer/ap-vector:0.40.2-1
+    images:
+      repository: quay.io/astronomer/ap-vector
+      tag: 0.40.1
```

Comment:
Why are we not going with 0.40.2-1 ?

Reply:
that we have not tested yet and we are not combining with this change